src/data/faq.json

```diff
@@ -542,20 +542,108 @@
+                                "title": "Why are certificates reissued?",
                                 "anchor": "dcc_replacement",
                                 "active": false,
                                 "textblock": [
-                                    "The EU specifications for booster vaccination certificates have changed. If your certificate no longer meets the current specifications, it is still valid, but it may not be recognized as a booster vaccination during an inspection. In such a case, you can be informed about it via the Corona-Warn-App version 2.20.3 and later with an appropriate message. You can then voluntarily request a new vaccination certificate free of charge via the app. After you have given your consent, the app transmits the vaccination certificate to the RKI for verification, which issues an updated certificate and transmits it to your app. You will then find the updated certificate in the certificate tab. The old certificate is automatically moved to the recycle bin and deleted after 30 days. The old certificate can be restored at any time within the 30 days.",
+                                    "Two cases can be distinguished:",
+                                    "(a) As of version 2.23, EU digital COVID certificates whose validity expires can be updated. Currently, EU digital COVID certificates issued in Germany have a technical expiration date of 365 days after issuance. Even if the corresponding certificate would still be technically valid, it will no longer be recognized as valid after reaching the technical expiration date. In June 2022, the first certificates will technically expire and can no longer be checked.",
+                                    "(b) The EU specifications for booster vaccination certificates have changed. If your certificate no longer meets the current specifications, it is still valid, but it may not be recognized as a booster vaccination during an inspection. In such a case, you can be informed about this via the Corona-Warn-App since version 2.20.3 with a corresponding message.",
                                     "For more information, visit the blog: <a href='../../blog/2022-03-30-cwa-2-20/' target='_blank'>Corona-Warn-App Version 2.20</a>"
-                                ]
-                            },
-                            {
-                                "title": "I receive a notification that the validity of a certificate is about to expire. What do I have to do?",
+                                ],
+                                "linkList": [
+                                    {
+                                        "title" : "Related FAQ articles:",
+                                        "links" : [
+                                            { "url" : "#dcc_replacement_howto", "text": "Wie und wann können Zertifikate in der App neu ausgestellt werden?"},

```

Comment:
This text should be English.

Same for all other links.

Reply:
It was a leftover of a reimplementation of the link list that I added yesterday evening. It's already changed and pushed.